modyn/trainer_server/internal/trainer/remote_downsamplers/abstract_remote_downsampling_strategy.py

```diff
@@ -50,8 +50,6 @@ def __init__(
         assert "downsampling_ratio" in params_from_selector
         self.downsampling_ratio = params_from_selector["downsampling_ratio"]
 
-        self.replacement = params_from_selector.get("replacement", True)
-
```

Comment:
sorry I have noticed this issue since a month ago then forgot to correct it :( otherwise everything has gone smoother